openid4vc-high-assurance-interoperability-profile-1_0.md

```diff
@@ -187,7 +187,7 @@ The following requirements apply to OpenID4VP, irrespective of the flow and Cred
 
 * The Wallet and Verifier MUST support at least one of the following Credential Format Profiles defined in (#vc-profiles): IETF SD-JWT VC or ISO mdoc. Ecosystems SHOULD clearly indicate which of these formats, IETF SD-JWT VC, ISO mdoc, or both, are required to be supported.
 * The Response type MUST be `vp_token`.
-* For signed requests, the Verifier MUST use, and the Wallet MUST accept the Client Identifier Prefix `x509_hash` as defined in Section 5.9.3 of [@!OIDF.OID4VP].
+* For signed requests, the Verifier MUST use, and the Wallet MUST accept the Client Identifier Prefix `x509_hash` as defined in Section 5.9.3 of [@!OIDF.OID4VP]. X.509 certificate profiles for use with `x509_hash` are out of scope of this specification. Ecosystems MAY define their own X.509 certificate profiles for `x509_hash` and use them accordingly. For example, an mDL ecosystem may use the Reader Authentication Certificate profile defined in ISO/IEC 18013-5, Annex B with `x509_hash`.
```

Comment:
I guess it was true anyway, but basically the implication is you probably can't request an (potentially non-EU) mDL and a EU PIID in a single traditional OID4VP request?

Reply:
On a meta-level, if the Wallet relies on a CA for RP certificates, the CA's issuance policies can provide those assurances.

Reply:
```suggestion
* For signed requests, the Verifier MUST use, and the Wallet MUST accept the Client Identifier Prefix `x509_hash` as defined in Section 5.9.3 of [@!OIDF.OID4VP]. X.509 certificate profiles to be used with `x509_hash` are out of scope of this specification. Ecosystems MAY define their own X.509 certificate profiles for `x509_hash` and use them accordingly. For example, an mDL ecosystem can use the Reader Authentication Certificate profile defined in ISO/IEC 18013-5, Annex B with `x509_hash`.
```